src/SqlClient/packages.config

```diff
@@ -1,4 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="FSharp.Compiler.Tools" version="10.0.1" targetFramework="net462" />
```

Comment:
This will break building this project with vs2015 as well as using this in F# 4.0 codebases and FSI.

Can you trim the PR to keep only the necessary code changes?

Reply:
I’m not sure why we discuss this PR here because the project is clearly abandoned by all the maintainers. I’ve built the package locally and my team just use it.

Reply:
I'm sure the project is being used in the wild (check download count on nuget, it is still in the first page among all fsharp tagged packages) so as long as we get one of the contributor to approve and merge the PR, I think we are in better place than having each user maintain own fork of the library.

I'm discussing this PR because I just reviewed a bunch of open PR, I'm also expecting another beta release which includes one of my merged PRs and possibly another.

To merge remaining PRs, I think we need to trim the extraneous changes which aren't needed and add unit tests covering the change.

Reply:
Firstly, we need a new maintainer. It's pointless to discuss anything in a dead repo. 

@dmitry-a-morozov @dsevastianov are you ok if I or @smoothdeveloper become a new maintainer (or better - both)?

Reply:
@vasily-kirichenko I'll add you as soon as I figure out how. It doesn't look like I have rights so far.

Reply:
@smoothdeveloper The project references FSharp.Core from GAC, it does not work on Mac at all, so we need to reference some version of the NuGet package. As I understand, you're against of using the latest version. Why? If you are right, what version we should use?